SpotifyInterface.c

```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
             break;
 
         case GetUserInfo:
+        //SEZ@Done ESP_LOGW just for test.
+        ESP_LOGW(TAG, "GetUserInfo");
```

Comment:
you want to keep this log? I think we print them later on top layer

Reply:
i want to show the other, this is the place of an error occurring, but I can remove them